components/Article/AuthorList/Author/__tests__/index.test.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import Author from '..';
+import { IntlProvider } from 'react-intl';
+
+describe('Author component', () => {
+  it('renders correctly', () => {
+    const username = 'test-author';
+    const { container } = render(
+      <IntlProvider locale="en">
```

Comment:
I forgot to mention, but @AugustinMauroy brought that to our attention, you might want to add `onError={() => {}}` so that the console.errors that are being emitted stop being emitted because of ReactIntl